pyproject.toml

```
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+exclude = ["htmlcov"] # Exclude the coverage report file from setuptools package finder
+
+[project]
+name = "WSIMOD"
+description = "WSIMOD is for simulating water quality and quantity"
+readme = "README.md"
+version = "0.2.0"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Imperial College London RSE Team", email = "ict-rse-team@imperial.ac.uk" },
+    { name = "Barnaby Dobson", email = "b.dobson@imperial.ac.uk" }
```

Maybe reverse the order of these. After all, Barney was the initiator...

```suggestion
    { name = "Barnaby Dobson", email = "b.dobson@imperial.ac.uk" },
    { name = "Imperial College London RSE Team", email = "ict-rse-team@imperial.ac.uk" }
```